Changelog.md

```diff
@@ -1,4 +1,7 @@
 # Unreleased
+- Fix `fossa container analyze --project` flag ([#238](https://github.com/fossas/spectrometer/pull/238)))
```

Comment:
[nit]: This should probably also mention the revision flag, since that was fixed as well.